cli/command/container/cp.go

```diff
@@ -448,3 +447,15 @@ func splitCpArg(arg string) (ctr, path string) {
 
 	return ctr, path
 }
+
+// IsAbs is a platform-agnostic wrapper for filepath.IsAbs.
+//
+// On Windows, golang filepath.IsAbs does not consider a path \windows\system32
+// as absolute as it doesn't start with a drive-letter/colon combination. However,
+// in docker we need to verify things such as WORKDIR /windows/system32 in
+// a Dockerfile (which gets translated to \windows\system32 when being processed
+// by the daemon). This SHOULD be treated as absolute from a docker processing
+// perspective.
+func isAbs(path string) bool {
+	return filepath.IsAbs(path) || strings.HasPrefix(path, string(os.PathSeparator))
+}
```

Comment:
Actually wondering now if we should share the implementation from the compose-loader, which has a more comprehensive implementation adopted from go stdlib; https://github.com/docker/cli/blob/91d097e70bc7a720f15ec67eda104f6d4faa52e9/cli/compose/loader/windows_path.go#L8-L27